database-mongodb/index.js

```diff
@@ -1,7 +1,22 @@
 const mongoose = require('mongoose');
+const Mockgoose = require('mockgoose');
+
+const mockgoose = new Mockgoose(mongoose);
 
 //development:
-mongoose.connect('mongodb://localhost/description_directions_attributes', { useUnifiedTopology: true, useNewUrlParser: true });
+if (process.env.node_env === 'test') {
+  mockgoose.prepareStorage().then(() => {
```

Comment:
Is this when the data you created (with the id of 50) gets put into the fake database?